### PR TITLE
Update license-header and pre-commit

### PR DESCRIPTION
**Pull request by kse3hi**
ADO#17300

### CODE REVIEW

**Review by mml5bg — DISMISSED** [review]
LGTM

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "dockerfile": "Dockerfile",
     // Update 'VARIANT' to pick a Node version: 12, 14, 16
     "args": {
-      "VARIANT": "16"
+      "VARIANT": "18"
```
Comment (MP91) [inline .devcontainer/devcontainer.json:9]: is that safe? did we properly test it?

Comment (MP91) [reply .devcontainer/devcontainer.json:9]: @dennismeister93 ?

**Review by MP91 — APPROVED** [review]
lgtm

**Pull request merged**